File: DirtBikeServer/DirtBikeServer/Models/Park.cs
namespace DirtBikeServer.Models {
    public class Park {
        public Guid Id { get; set; } = Guid.NewGuid();
        public string Name { get; set; } = String.Empty;
        public string Location { get; set; } = String.Empty;
        public string? Description { get; set; } 
        public decimal AdultPrice { get; set; }
        public decimal ChildPrice { get; set; }
        public int GuestLimit { get; set; }
        public virtual ICollection<Booking> Bookings { get; set; } = new List<Booking>();
    }
}

File: src/models/park.ts
import Review from "./review";

// You can use whatever data type for ID as long as it's guaranteed unique
// Here we use GUIDs because they're almost guaranteed unique
// and any other id system is abnormal and honestly worse in my opinion
export default interface IPark {
    parkName: string;
    id: string;
    location: string;
    description: string;
    reviews: Review[];
    imageUrl?: string;
    adultPrice: number;
    childPrice: number;
}

export interface ApiPark {
    id: string;
    name: string;
    location: string;
    description: string | null;
    adultPrice: number;
    childPrice: number;
    guestLimit: number;
    bookings?: Review[];
}